SDC-Sedeco/Atelier
# Switch product service endpoint

## Description

Switch API calls for products and styles from the legacy API to the new product overview service.

## How to test

It won't work on other machines locally since product data doesn't exist in your database. 
But, assuming that it will work: 
1. Pull this branch, `npm install`, `npm run server-dev` and `npm run client-dev` to start up the app
2. Pull [front-end-integration](https://github.com/SDC-Sedeco/Product-Overview-Service/tree/front-end-integration) branch from product overview service, `npm install` and `npm start` to start up the service
3. Go to http://localhost:3000/59553 and verify that the full app should render properly (product data is from the new service, other data is from legacy API)
4. Go to http://localhost:3000/1 and verify that only product overview would render properly (legacy API doesn't start from product id 1, so doesn't have data for other components)

## Notes

I'll hold off on merging this PR until we are ready to integrate all our services together.

## Issue tracking

https://trello.com/c/dzymoEpu
